mods/aero-tray.wh.cpp

```diff
@@ -16,6 +16,11 @@
 This mod restores the tray overflow from Windows 7 and 8, including the "Customize..." link,
 drawing tray icons with the system theme, and the general appearance.
 
+# IMPORTANT: READ!
+This mod needs to hook into `winlogon.exe` to successfully capture Explorer starting. Please
```

Comment:
To be more persico, it should say "Windhawk needs to hook into `winlogon.exe`...".